### PR TITLE
25 add suggested start date content

### PR DESCRIPTION
**Pull request by PaulBredl**
## Description of changes
Add suggested date to content. This is the date when the content should be done. Frontend should probably use the suggested   start date of the chapter as a default value.
Add initial learning interval to assessments. Default is 1 day. For some types of assessments a higher initial learning interval makes sense.
  
## How has this been tested?
Please describe the test strategy you followed.

- [x] automated unit test
- [x] automated integration test
- [ ] automated acceptance test
- [ ] manual, exploratory test
    
## Checklist before requesting a review
- [x] My code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [ ] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code fulfills all acceptance criteria
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] The test coverage (line and branch) is reasonably high, especially on Service classes and other classes with complex logic
- [ ] I have made corresponding changes to the documentation
- [ ] I have added explanation of architectural decision and rationales to [wiki/adr](https://github.com/IT-REX-Platform/wiki/tree/main/adr)
- [ ] I have updated the changes in the ticket description

## Checklist for reviewer
- [ ] The code works and does not throw errors
- [ ] The code is easy to understand and there are no confusing parts
- [ ] The code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [ ] The code change accomplishes what it is supposed to do
- [ ] I cannot think of any use case in which the code does not behave as intended
- [ ] The added and existing tests reasonably cover the code change
- [ ] I cannot think of any test cases, input or edge cases that should be tested in addition
- [ ] Description of the change is included in the documentation

### CODE REVIEW

File: src/test/java/de/unistuttgart/iste/gits/content_service/api/mutation/MutationCreateMediaContentTest.java [hunk 1]
```diff
@@ -1,69 +1,74 @@
...
 import java.util.UUID;
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
 
 @ContextConfiguration(classes = MockTopicPublisherConfiguration.class)
 @GraphQlApiTest
+@TablesToDelete({"content_tags", "content", "tag"})
```
Comment (PaulBredl) [inline src/test/java/de/unistuttgart/iste/gits/content_service/api/mutation/MutationCreateMediaContentTest.java:31]: This is necessary because the logic "just delete all tables after each test" doesn't work with the @ManyToMany annotation. To fix it, the order of the tables must be set correctly, otherwise, there will be a constraint violation.

**Review by myluki2000 — APPROVED** [review]
lgtm
